### PR TITLE
[DatePicker] add props to set datepicker action bar buttons title

### LINKED ISSUE #2861: Can't translate the actionBar in the datePicker component (blueprint 2 and 3)
#### Environment

- __Package version(s)__: 2 and 3
- __Browser and OS versions__: Ubuntu 18.04 Chrome  68.0.3440.106 (Official Build) (64-bit)

#### Steps to reproduce

1. Activate the showActionBar option in a datepicker
2. Give the fr-CA locale as props to the datepicker

#### Actual behavior

The buttons are in english and we have no way to translate them.

#### Expected behavior

The buttons in french or we can override the text in the datePicker.tsx in function renderOptionsBar.

`private renderOptionsBar() {
        return (
            <div className={Classes.DATEPICKER_FOOTER}>
                <Button className={CoreClasses.MINIMAL} onClick={this.handleTodayClick} text="Today" />
                <Button className={CoreClasses.MINIMAL} onClick={this.handleClearClick} text="Clear" />
            </div>
        );
    }`

#### Possible solution

Either have a locale file in french that we take this from and we can override or maybe just add two functions to the localeUtils that we can provide to the component.

### PR DESCRIPTION
**Pull request by NachoJusticia**
#### Changes proposed in this pull request:
Add `todayButtonText` and `clearButtonText` props to DatePicker component so these titles can be set from outside.

#### Reviewers should focus on:
They way `todayButtonText` and `clearButtonText` props has been added to datePicker component and its jsdoc comments.

#### Screenshot
captura de pantalla 2018-09-14 a las 16 51 18

### CODE REVIEW

File: packages/datetime/src/datePicker.tsx [hunk 1]
```diff
@@ -55,6 +55,18 @@ export interface IDatePickerProps extends IDatePickerBaseProps, IProps {
...
     showActionsBar?: boolean;
 
+    /**
+     * Today action bar button title.
+     * @default "Today"
+     */
+    todayButtonText?: string;
+
+    /**
+     * Reset action bar button title.
```
Comment (giladgray) [inline packages/datetime/src/datePicker.tsx]: weird to say `title` here and `Text` in the prop name. consistency please. same for the prop above.

> Text for the reset button in the action bar.

**Comment by giladgray** [comment]
@NachoJusticia have you signed the CLA? sometimes it goes yellow on its own so just checking.

**Comment by NachoJusticia** [comment]
Hi @giladgray 
Yes, I signed the CLA last week

**Comment by giladgray** [comment]
@NachoJusticia ok cool. mind pushing an empty commit to trigger an update?

**Comment by NachoJusticia** [comment]
Hi again @giladgray , I have just pushed an empty commit as you suggested me and now cla step is passing 👍

**Pull request merged**